.github/workflows/deploy.yml

```diff
@@ -41,4 +85,27 @@ jobs:
       - name: Install SAM CLI
```

Comment:
Dont we need to use the beta aws access and security key above?

Reply:
fixed